syntax = "proto2";
package com.protobuf.msg;
import  "Status.proto";

message Message {
	optional msgType type = 1;
	optional string msgBody = 2;
}